simphony_ui/couple_CFDDEM.py

```diff
@@ -25,38 +24,32 @@
 
 def run_calc(output_path, mesh_name):
     """Executes the full calculation"""
-    runstart = time.time()
-
-    mode_OF = "internal"
+    mode_cfd = "internal"
     mesh_type = "block"
 
     # Defining the wrapper for OpenFoam
-    if mode_OF == "internal":
-        wrapper_OF = openfoam_internal.Wrapper()
-        CUBAExt_OF = openfoam_internal.CUBAExt
-    elif mode_OF == "io":
-        wrapper_OF = openfoam_file_io.Wrapper()
-        CUBAExt_OF = openfoam_file_io.CUBAExt
+    if mode_cfd == "internal":
```

Comment:
This is more like openfoam_params_mode. start partitioning it out in "model class"-like things, so making it a class it's just changing from `openfoam_params_mode` to `openfoam_params.mode`